README.md

```diff
@@ -323,6 +323,35 @@ Try and leave this code a little better than you found it.
 Non-explicity versions like `~0.2.1` can lead to different behavior from one test run to the next if that dependency
 version changes.
 
+## The Mike Ratio
+
+John Boxall is an advocate for small teams, largely because they are able to move
+very quickly. Another great way to improve your team's velocity is to hire at least
+one Mike.
+
+For millennia, Mikes (or Michaels) have been known to be highly efficient, top performers.
+Consider the following list of Michaels:
+
+- Michael Phelps
+- Michelangelo
+- Michael Jordan
+- Michael Caine
+- Mike D
```

Comment:
Also, Michael Palin, which supports the theory that great teams should have a Michael (ie. the Pythons)